arch/arm/src/stm32/stm32_tickless.c

```diff
@@ -267,7 +268,7 @@ static int stm32_tickless_setchannel(uint8_t channel)
 
   /* Set polarity */
 
-  ccer_val |= ATIM_CCER_CC1P << (channel << 2);
+  ccer_val |= ATIM_CCER_CC1P << GTIM_CCER_CCXBASE(channel);
```

Comment:
I get that the macro is the same, but why not use `ATIM_CCER_CCXBASE` here?

Reply:
@btashton I just pushed an update to replace even more ATIM_/BTIM_ macros with GTIM_ macros for consistency. There are now no uses of BTIM_ macros in the codebase - which makes sense since all our drivers focus on General timer implementation with some extended features for Advanced timers. I've left the use of ATIM_ anywhere we were actually meaning to use advanced features.

I'm sure I missed some things.